ckanext/datapusher/logic/action.py

```diff
@@ -158,6 +168,9 @@ def datapusher_hook(context, data_dict):
     task['state'] = status
     task['last_updated'] = str(datetime.datetime.now())
     if status == 'complete':
+        for plugin in p.PluginImplementations(interfaces.IDataPusher):
```

Comment:
I would move the hook a few lines below and pass the `resource _dict` and `dataset_dict` as params to the plugins, to save them the calls